### PR TITLE
Damage type standardizations SKILL part 2/2

### PR DESCRIPTION
**Pull request by terrapin47**
Change damage types to Pierce/Slash/Strike
Replacing instances of Hit, Impact, and Stab.

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -5234,7 +5234,7 @@ SKILL_20150918_005233	$Cloaking: Movement Speed Increase
 SKILL_20150918_005234	$Increases the movement speed of [Cloaking] by 1 per attribute level.
 SKILL_20150918_005235	
 SKILL_20150918_005236	
-SKILL_20150918_005237	$Deals 10% additional damage to Plant-type monsters when equipped with a [Blunt].
+SKILL_20150918_005237	$Deals 10% additional damage to Plant-type monsters when equipped with a [Bludgeon] type weapon.
```
Comment (Sunaries) [inline SKILL.tsv]: Reverse the change since this description is talking about a weapon type.

File: SKILL.tsv [hunk 2]
```diff
@@ -4819,7 +4819,7 @@ SKILL_20150717_004818	According to skill 5 seconds at MakuTaro bursts was 10% of
 SKILL_20150717_004819	The enemies that are shot by [Magic Arrow] will fall into [Silence] with a certain probabillity. The probability will increase by 10% per Attribute level.
 SKILL_20150717_004820	You can create the arrows for Fletcher. Crafting arrow is available while rest mode.
 SKILL_20150717_004821	{memo X}Defense decrease effect of [Deprotected Zone] increases by 1 per attribute level.
-SKILL_20150717_004822	$Increases the chance of stun by 2% per attribute level when using a basic attack on enemies with a [Blunt] weapon type.
+SKILL_20150717_004822	$Increases the chance of stun by 2% per attribute level when using a basic attack on enemies with a [Bludgeon] weapon type.
```
Comment (Sunaries) [inline SKILL.tsv]: Reverse the change since this description is talking about a weapon type.

File: SKILL.tsv (hunk 1 above)
Comment (Sunaries) [inline SKILL.tsv]: This is talking about the weapon type, I don't know if its called [Blunt] or [Blunt Instrument].

File: SKILL.tsv (hunk 2 above)
Comment (Sunaries) [inline SKILL.tsv]: This is talking about the weapon type, I don't know if its called [Blunt] or [Blunt Instrument].

File: SKILL.tsv [hunk 3]
```diff
@@ -1982,7 +1982,7 @@ SKILL_20150317_001981	$Adds additional damage to frozen enemies when attacking w
 SKILL_20150317_001982	$Rupture Blow
 SKILL_20150317_001983	Has a chance of inflicting damage with an explosion on nearby enemies when attacking with Hit-attacks. (5% chance of activating per level)
 SKILL_20150317_001984	$Rupture
-SKILL_20150317_001985	$Increases damage when using a Blunt weapon on chain armor-type enemies.{nl}(Increases by 20% per level)
+SKILL_20150317_001985	$Increases damage when using a Bludgeon weapon on Plate armor-type enemies.{nl}(Increases by 20% per level)
```
Comment (Sunaries) [inline SKILL.tsv]: This is talking about the weapon type, I don't know if its called [Blunt] or [Blunt Instrument].

**Comment by terrapin47** [comment]
Updated as per feedback on discord.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**